packages/core/useVirtualList/index.ts

```diff
@@ -119,21 +119,24 @@ function useVirtualListResourses<T>(list: MaybeRef<T[]>): UseVirtualListResource
   return { state, source, currentList, size, containerRef }
 }
 
-function createGetViewCapacity<T>(state: UseVirtualListResources<T>['state'], source: UseVirtualListResources<T>['source'], itemSize: UseVirtualListItemSize) {
-  return (containerHeight: number) => {
+function createGetViewCapacity<T>(
+  state: UseVirtualListResources<T>['state'],
+  source: UseVirtualListResources<T>['source'],
+  itemSize: UseVirtualListItemSize,
+) {
+  return (containerSize: number) => {
     if (typeof itemSize === 'number')
-      return Math.ceil(containerHeight / itemSize)
+      return Math.ceil(containerSize / itemSize)
```

Comment:
Height naming was from when list was vertical only. Size is more appropriate, IMO